### ISSUE TITLE
Add advice to improve documentation that doesn't pass checks

### ISSUE THREAD

**Issue by thisisnic** [thisisnic/docreview#39]
`goodpractice` adds advice for fixing failed checks - add the functionality for doing that here. For example, if you have a check for if images have alt text, and that check is failed, print out links to resources which tell you how to add alt text to your images.
